ui/js/component/transactionList/internal/TransactionListItem.jsx

```diff
@@ -6,12 +6,59 @@ import Link from "component/link";
 import lbryuri from "lbryuri";
 
 class TransactionListItem extends React.PureComponent {
+  abandonClaim() {
+    const {
+      claim_id: claimId,
+      claim_name: name,
+      txid,
+      type,
+      nout,
+    } = this.props.transaction;
+    let msg;
+
+    if (type == "tip") {
+      msg = "This will reduce the committed credits to the URL";
```

Comment:
The determination of the message can be put in the modal itself.

Reply:
Modal body:

Are you sure you want to unlock these credits?

These credits are permanently yours and can be unlocked at any time. Unlocking them allows you to spend them, but can hurt the performance of your content in lookups and search results. It is recommended you leave tips locked until you need or want to spend them.

Reply:
Done.